examples/fibonacci.rs

```diff
@@ -163,3 +173,17 @@ fn main() {
         }
     }
 }
+
+#[cfg(test)]
+#[test]
+fn test_pil_single_circuit_fibo() {
```

Comment:
Does it make sense to add a test to an example? shouldn't it be in the main?

Reply:
Agree and fixed!